db/migrate/20231201194409_create_form_submissions.rb

```diff
@@ -0,0 +1,18 @@
+class CreateFormSubmissions < ActiveRecord::Migration[6.1]
+  def change
+    create_table :form_submissions do |t|
+      t.integer :form_type, default: 0, null: false
+      t.string :benefits_intake_uuid
+      t.string :submitted_claim_uuid
```

Comment:
I don't know what difference (if any) this makes under the hood but can these `t.string`s be `t.uuid`s?